aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -30,3 +30,15 @@ proposals:
           transaction_shuffler_type:
             deprecated_sender_aware_v1: 32
           block_gas_limit : 35000
+  - name: enable_enable_storage_deletion_refund
+    metadata:
+      title: "Enable Storage Deletion Refund"
+      description: "AIP-32: Storage Deletion Refund"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/127"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
```

Comment:
Is there any initialize function that needs to be called? I thought I saw one

Reply:
ugh....

Reply:
So it needs to be a `RawScript`?

Reply:
updated